packs/feat-effects/effect-bon-mot.json

```diff
@@ -16,19 +16,34 @@
             "value": 1
         },
         "publication": {
-            "license": "OGL",
-            "remaster": false,
-            "title": "Pathfinder Advanced Player's Guide"
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Player Core 2"
         },
         "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "PF2E.Check.Result.Degree.Check.criticalSuccess",
+                        "value": -3
```

Comment:
Since the effect will have outcome roll options when used from the macro, might be good to take the chance to add predicates here to autopick the correct degree of success if that's the case, the same way Overdrive and other effects do

Reply:
While it's a good idea, it doesn't look like the outcome roll options are available.

Reply:
![image](https://github.com/user-attachments/assets/b3e851c0-1d95-419e-b45e-b0a4f30ace57)
Dragging from the chat card after rolling from the macro

Reply:
Ah, thanks. I just inspected the roll itself. I'll try it out.